npm/pkg/dataplane/dataplane.go

```diff
@@ -24,9 +24,12 @@ type PolicyMode string
 type Config struct {
 	*ipsets.IPSetManagerCfg
 	*policies.PolicyManagerCfg
+	// helpful for UTs (defaults to false for external packages)
+	disableGoRoutines bool
```

Comment:
can we rename this to something like `concurrencyEnabled` or `concurrencyDisabled`

Reply:
Looking at what it does below - maybe something like `disableReconcile`?